Dockerfile

```diff
@@ -27,8 +27,9 @@ ENV PATH=$GOPATH/bin:/usr/local/go/bin:$PATH
 
 # Download Go.
 ARG GO_VERSION=1.18
-ARG TARGETPLATFORM
-RUN export ARCH=${TARGETPLATFORM//\//-} && curl -o go.tgz "https://dl.google.com/go/go${GO_VERSION}.${ARCH}.tar.gz" \
+ARG TARGETARCH
```

Comment:
Can we possibly set some common defaults like amd64, so that for those who don't enable buildkit the build does not fail?